scripts/training/configs/paper-config-tiny.yaml

```diff
@@ -0,0 +1,35 @@
+training_data_paths:
```

Comment:
For these config files, can we have the full model name as part of the file names? `*chronos-t5-tiny*.yaml`